compiler-rt/lib/rtsan/rtsan_checks.inc

```diff
@@ -17,3 +17,4 @@
 // SummaryKind should be a string literal.
 
 RTSAN_CHECK(CallStackContains, "call-stack-contains")
+RTSAN_CHECK(FunctionNameIs, "function-name-is")
```

Comment:
Please review the new name @davidtrevelyan and @fmayer  - I chose one with a verb at the end to mirror call-stack-contains.

Other candidates I considered:
* unsafe-function-name-is
* function-name
* function-name-matches

Reply:
I like `function-name-is` if the intention is only to allow users to provide an exact function name match, character for character. But I think that would be difficult to use - how is the user supposed to reason about namespaces, mangling, parentheses on the end, overloads, etc.? I think at the minute I prefer `function-name-matches` with regex supported.

Reply:
Cool that works for me. At least as written, we use the suppression built-in simple regexes (which I think is good, and we should keep it). I'll wait to see what fmayer says.

Reply:
Sounds good!

Reply:
`function-name-matches` SGTM if it's a regex / glob